src/executor/drop_executor.cpp

```diff
@@ -10,9 +10,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include <include/catalog/index_catalog.h>
 #include "executor/drop_executor.h"
 
+#include "include/catalog/index_catalog.h"
```

Comment:
`"include/catalog/index_catalog.h"` -> `"catalog/index_catalog.h"`

Sorry for being pedantic. The new formatter from @tcm-marcel will do this automatically (#991)